acceptance/tests/resource/user/should_create_with_expiry.rb

```diff
@@ -0,0 +1,30 @@
+test_name "verifies that puppet resource creates a user and assigns the correct expiry date when absent"
+confine :except, :platform => 'windows'
+confine :except, :platform => /^eos-/ # See ARISTA-37
+confine :except, :platform => /^cisco_/ # See PUP-5828
+
+tag 'audit:high',
+    'audit:refactor',  # Use block style `test_run`
```

Comment:
Please use block style and remove `'audit:refactor'`

```ruby
test_name '' do
  step '' do
  end
end
```

Also i think after the functionality is tested, this can be moved de medium and useradd covered with unit tests as i don't think that the system command will change, we only need to ensure that we call it with the correct parameters